client/src/components/steam-user.js

```diff
@@ -6,10 +6,14 @@ export default function (props) {
   return (
     <Link to={`/search/${props.steamUser.id}`}>
       <Media className="align-items-center">
-        {props.steamUser.avatar && (
+        {props.steamUser.avatar ? (
           <span className="avatar avatar-sm rounded-circle">
             <img alt="..." src={props.steamUser.avatar} />
           </span>
+        ) : (
+          <span className="avatar avatar-sm rounded-circle">
+            <img alt="..." src={require('../assets/img/misc/avatar.svg')} />
```

Comment:
Could you do `import avatar from ...;` and then `src={avatar}`, please? This does not work with newer versions of Nodejs so I've had to replace the rest of these in the existing code.